extensions/guacamole-auth-json/src/main/java/org/apache/guacamole/auth/json/connection/ConnectionService.java

```diff
@@ -176,6 +177,22 @@ public GuacamoleTunnel connect(UserData.Connection connection,
         String hostname = proxyConfig.getHostname();
         int port = proxyConfig.getPort();
 
+        // handle guacd-[hostname/port/ssl] overrides
+        Map<String, String> params = connection.getParameters();
```

Comment:
The guacd hostname, port, etc. are not connection parameters, which are strictly the protocol-specific key/value pairs for configuring the connection as defined by the protocol plugin that guacd ultimately loads. Additional properties configuring the details for guacd will need to be at a different level of the JSON, perhaps directly within the connection, not within the connection params.

Reply:
Hi Mike. I Just pushed an other commit migrating this to connection level.

Currently i'm not sure about the best approach - I've allinged the java class with the current camelCase, so the settings are currently also in camelCase:

```
{
    "username" : "arbitraryUsername",
    "expires" : TIMESTAMP,
    "connections" : {

        "Connection Name" : {
            "protocol" : "PROTOCOL",
            "guacdHostname" : "my-other-guacd-host",
            "guacdPort" : 4822,
            "guacdSsl" : false,
            "parameters" : {
                "name1" : "value1",
                "name2" : "value2",
                ...
            }
        },
        ...
    }
}
```
But that's not matching the names in other places. I could set some annotations for the jackson data mapper, but haven't found any json annotation on the code so i'm not sure about this...